http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpWsmDao.scala

```diff
@@ -151,6 +153,43 @@ class HttpWsmDao[F[_]](httpClient: Client[F], config: HttpWsmDaoConfig)(implicit
       )(onError)
     } yield res
 
+  override def getLandingZone(billingProfileId: String, authorization: Authorization)(implicit
+    ev: Ask[F, AppContext]
+  ): F[Option[LandingZone]] =
+    for {
+      ctx <- ev.ask
+      res <- httpClient.expectOptionOr[ListLandingZonesResult](
+        Request[F](
+          method = Method.GET,
+          uri = config.uri
+            .withPath(
+              Uri.Path
+                .unsafeFromString(s"/api/landingzones/v1/azure?billingProfileId=${billingProfileId}")
+            ),
+          headers = headers(authorization, ctx.traceId, withBody = false)
+        )
+      )(onError)
+      landingZoneOption = res.flatMap(listLandingZoneResult => listLandingZoneResult.landingZones.headOption)
+    } yield landingZoneOption
+
+  override def listLandingZoneResourcesByType(landingZoneId: UUID, authorization: Authorization)(implicit
+    ev: Ask[F, AppContext]
+  ): F[Option[List[LandingZoneResourcesByPurpose]]] =
```

Comment:
Feels like we can use `expectOr` instead of `expectOptionOr` and return a `F[List[LandingZoneResourcesByPurpose]]`.